vaadinfrontend/frontend/themes/datamanager/components/custom.css

```diff
@@ -9,3 +9,37 @@
 @import "page-area.css";
 @import "span.css";
 @import "vaadin-custom.css";
+
+.card-deck {
+  display: flex;
+  align-content: space-evenly;
+  flex-flow: row wrap;
+}
+
+.navigation-button {
+  display: flex;
+  flex-direction: column;
+  gap: var(--lumo-space-m);
+  align-items: center;
+  flex: 1 1;
+}
+
+.navigation-button .button {
+  box-shadow: var(--lumo-box-shadow-xs);
+}
+
+.navigation-button .label {
+  font-size: var(--lumo-font-size-s)
+}
+
+.error-text {
+  color: var(--lumo-error-text-color);
+}
+
+vaadin-multi-select-combo-box::part(toggle-button)::before {
+  color: var(--lumo-primary-color);
+}
+
+vaadin-number-field::part(decrease-button), vaadin-number-field::part(increase-button) {
+  color: var(--lumo-primary-color);
+}
```

Comment:
Can be removed completely since these classes are declared in their respective css files(button.css, div.css, vaadin-custom.css, span.css)
```suggestion
```